go/ct/st/memory.go

```diff
@@ -83,9 +83,23 @@ func (m *Memory) Grow(offset, size uint64) {
 	}
 }
 
+const (
+	// Maximum memory size allowed
+	// This magic number comes from 'core/vm/gas_table.go' 'memoryGasCost' in geth
```

Comment:
Please provide a cleaner definition for this value. Saying that it is a "magic number from some source" is not building up trust.